examples/binning/binning.py

```diff
@@ -7,7 +7,7 @@
 import pandas as pd
 from winton_kafka_streams.processor import BaseProcessor, TopologyBuilder
 import winton_kafka_streams.kafka_config as kafka_config
-import winton_kafka_streams.kafka_stream as kafka_stream
+import winton_kafka_streams.kafka_streams as kafka_streams
```

Comment:
I'm not sure what you mean exactly. This link on master shows that the package name, module name and class name are all plural now:

https://github.com/wintoncode/winton-kafka-streams/blob/master/winton_kafka_streams/kafka_streams.py#L14